.travis.yml

```diff
@@ -20,6 +20,7 @@ matrix:
         - sudo add-apt-repository 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise-3.9 main' -y
         - sudo apt-get update -q
         - sudo apt-get install clang-3.9 llvm-3.9 llvm-3.9-runtime libunwind8-dev -y
+        - sudo pip install virtualenv
```

Comment:
The sudo is doing pip outside of the virtualenv that the build is in for Travis CI.

Reply:
Oops, also forgot to point out it also needs presumably the same addition to before_install: in the other, non-g++-6 environment.

Travis CI config is "fun".